R/plugin-providers.R

```diff
@@ -47,7 +61,7 @@ addProviderTiles <- function(
 #' errorTileUrl,noWrap,opacity,zIndex,updateWhenIdle,detectRetina
 #' the tile layer options; see
 #' <https://web.archive.org/web/20220702182250/https://leafletjs.com/reference-1.3.4.html#tilelayer>
-#' @param ... named parameters to add to the options
+#' @param ... For [providerTileOptions()], named parameters to add to the options. Not used in [addProviderTiles()].
```

Comment:
I was trying to require that users provide a named argument. But given it's so far into the args, we should remove the `...`.  

I'll remove it as I had suggested it.